default/widgets.py

```diff
@@ -0,0 +1,6 @@
+from django.forms.widgets import Widget
+
+
+class Display(Widget):
+    def render(self, name, value=None, attrs=None):
+        return '<span>{}</span>'.format(value)
```

Comment:
Did you check that autocompletion of email/password still works with a `span`, maybe a non editable (`readonly`) input is more appropriated?